pkg/virt-operator/webhooks/kubevirt-update-admitter.go

```diff
@@ -432,6 +436,20 @@ func validateInfraReplicas(replicas *uint8) []metav1.StatusCause {
 	return statuses
 }
 
+func validateRolloutStrategy(strategy *v1.VMRolloutStrategy) []metav1.StatusCause {
+	var statuses []metav1.StatusCause
+
+	if (strategy.LiveUpdate == nil && strategy.Stage == nil) ||
+		(strategy.LiveUpdate != nil && strategy.Stage != nil) {
+		statuses = append(statuses, metav1.StatusCause{
+			Type:    metav1.CauseTypeFieldValueInvalid,
+			Message: "vmRolloutStrategy must contain exactly one element",
+		})
+	}
```

Comment:
I followed the style of the rest of the file. Besides, we need to make the item into a list and return the empty list otherwise, not much of an improvement...